corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -423,8 +424,12 @@ hqDefine("geospatial/js/geospatial_map", [
             });
         };
 
-        var missingGPSModel = function(cases) {
+        var missingGPSModel = function (cases, users) {
             this.casesWithoutGPS = ko.observable(cases);
+            this.usersWithoutGPS = ko.observable(users);
+        };
+        var missingGPSModelInstance = new missingGPSModel([], []);
```

Comment:
where is this being used?

Reply:
Found it.

Reply:
Looks like the only place this is initialized is here where we are not passing any cases or users, so seems okay to let go of the cases and users arguments in the function.

Reply:
Is it needed to create a new model for storing these lists? I guess we could instead just keep them on the view model?

Reply:
> Is it needed to create a new model for storing these lists?

This model gets binded to the DOM when `loadCases` or `loadUsers` is called. Given that, we can keep it as a model to make use of KO's bindings.

I do agree however, that we don't need to pass any arguments when initiailizing this model, so I'll remove those.

Reply:
[9a1c523](https://github.com/dimagi/commcare-hq/pull/33488/commits/9a1c52397c97edab7c831fc760de89c0ab298328)